src/layout/Group/DisplayGroupContainer.tsx

```diff
@@ -52,20 +47,45 @@ export function DisplayGroupContainer({ groupNode, id, onlyRowIndex, renderLayou
       data-testid='display-group-container'
       data-componentid={container.id}
     >
-      {title && (
+      {(title || texts.body) && (
```

Comment:
Btw, I just checked, and found no apps that already define a `body` text resource bindings on a `Group` component, so this should not break any existing apps. 🥳

Just for reference, I checked by adding this snippet to the existing one in `all.test.ts`:
```typescript
    for (const layout of Object.values(layouts)) {
      for (const component of layout || []) {
        if (component.type.toLowerCase() === 'group' && (!component.maxCount || component.maxCount <= 1)) {
          expect(component.textResourceBindings?.body).toBeUndefined();
        }
      }
    }
```

Reply:
Group with panel already has a body so that is a little surprising

Reply:
Yup, I agree.. 🤔 Maybe nobody has started actually using that, though - it was only re-introduced lately. I updated all the apps to make sure, but still couldn't find any such configuration - so maybe it's just because nobody has released any such thing on tt02 or prod.. I actually found no non-repeating groups with both `panel` and `textResourceBindings` at all. 🤷